functions.php

```diff
@@ -210,3 +211,23 @@ function ( $hidden ) {
 	10,
 	1
 );
+
+// Might move later.
+function register_more_blocks() {
+	register_block_type( 'p4/reading-time', [
```

Comment:
Since it's easy to add many back end only blocks, it's a good time to reconsider our block namespace. `planet4-blocks` is very long and the suffix `-blocks` is redundant. Using `p4/` instead saves us a lot of typing and long lines of code, and it's already a commonly known abbreviation in the project.